src/index.js

```diff
@@ -27,6 +65,42 @@ const products = [
 // 7. Target the button in index.html, attach an event listener to it. Every time we click
 // on the button, it will toggle the product list. Show => hide, hide => show
 
+const ulist = document.querySelector(".product-list");
+const btnToggle = document.querySelector(".toggle");
+var searchInput = document.querySelector(".search-input");
+const res = document.querySelector(".result")
+const form = document.querySelector(".product-form")
+btnToggle.addEventListener('click', function (){
+  products.forEach((element)=>{
+    const newLi = document.createElement('li')
+    newLi.innerText = element.name;
+    ulist.appendChild(newLi)
+  })
+  btnToggle.style.visibility = 'visible'? 'hidden': 'visible';
```

Comment:
Doesn't this make the button disappear? You won't be able to hide the list of products once it's displayed.

Reply:
yes you are right.... i will go thru it again... thanks